purefa/manifest.json

```
@@ -0,0 +1,67 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a2d8f393-62cd-4ece-bfab-e30797698b12",
+  "app_id": "purefa",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::OS System",
+    "Category::Data Store",
+    "Offering::Integration"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor the performance and utilization of Pure Storage FlashArrays",
+    "title": "Pure Storage FlashArray",
+    "media": [
+      {
+        "caption": "Pure Storage FlashArray - Overviewi (Part 1)",
+        "image_url": "images/FA-overview-1.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Pure Storage FlashArray - Overviewi (Part 2)",
+        "image_url": "images/FA-overview-2.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Pure Storage FlashArray - Overviewi (Part 3)",
```

```suggestion
        "caption": "Pure Storage FlashArray - Overview (Part 3)",
```
Same comment as above

---

Ack